commit 079e6c2b04faa8cd09478662a255cc87acfaede1
Author: Tobias Brunner <tobias@strongswan.org>
Date:   Wed Dec 11 17:24:03 2013 +0100

    pki: Increase MAX_COMMANDS to cover all currently available commands
    
    Fixes #452.

diff --git a/src/pki/command.h b/src/pki/command.h
index 1a884fb..3e66a1a 100644
--- a/src/pki/command.h
+++ b/src/pki/command.h
@@ -22,9 +22,9 @@
 #define COMMAND_H_
 
 /**
- * Maximum number of commands.
+ * Maximum number of commands (+1).
  */
-#define MAX_COMMANDS 10
+#define MAX_COMMANDS 11
 
 /**
  * Maximum number of options in a command (+1)
